core/common/src/main/java/alluxio/security/authentication/ChannelAuthenticator.java

```diff
@@ -174,9 +174,9 @@ public void authenticate() throws AlluxioStatusException {
           code = ((AlluxioStatusException) e).getStatusCode();
         }
         String message = String.format(
-            "Channel authentication failed with code:%s. ChannelKey: %s, AuthType: %s, Error: %s",
-            code.name(), mChannelKey.toStringShort(), mAuthType, e.toString());
-        LOG.warn(message);
+            "Channel authentication failed with code:%s. ChannelKey: %s, AuthType: %s",
+            code.name(), mChannelKey.toStringShort(), mAuthType);
+        LOG.warn(message, e);
```

Comment:
We generally avoid attaching the exception at the `WARN` level since stacktraces are scary.

Reply:
Reverted.